api/pom.xml

```diff
@@ -232,6 +254,7 @@
                             Jakarta Enterprise Beans ${spec.version} API Design Specification
                         </Bundle-Description>
                         <Specification-Vendor>${project.organization.name}</Specification-Vendor>
+                        <Multi-Release>true</Multi-Release>
```

Comment:
Do we really need this?

What I've heard is that multi release jar is only worth the engineering effort if you need different versions of the same class on different version of the java runtime.  I don't believe that is an issue for the EJB API jar. The Java 11 compiled `module-info.class` file can be placed at the root of the JAR and will be ignored by Java 8, and there are no other classes that will have java dependent versions.